src/main/java/nl/knaw/dans/easy/sword2examples/Common.java

```diff
@@ -194,21 +194,30 @@ public static CloseableHttpResponse sendChunk(DigestInputStream dis, int size, S
         byte[] chunk = readChunk(dis, size);
         String md5 = new String(Hex.encodeHex(dis.getMessageDigest().digest()));
         HttpUriRequest request = RequestBuilder.create(method).setUri(uri).setConfig(RequestConfig.custom()
-        /*
-         * When using an HTTPS-connection EXPECT-CONTINUE must be enabled, otherwise buffer overflow may follow
-         */
-        .setExpectContinueEnabled(true).build()) //
+            /*
+             * When using an HTTPS-connection EXPECT-CONTINUE must be enabled, otherwise buffer overflow may follow
+             */
+            .setExpectContinueEnabled(true).build()) //
                 .addHeader("Content-Disposition", String.format("attachment; filename=%s", filename)) //
                 .addHeader("Content-MD5", md5) //
                 .addHeader("Packaging", BAGIT_URI) //
                 .addHeader("In-Progress", Boolean.toString(inProgress)) //
                 .setEntity(new ByteArrayEntity(chunk, ContentType.create(mimeType))) //
                 .build();
+        addXAuthorizationToRequest(request);
         CloseableHttpResponse response = http.execute(request);
```

Comment:
shouldn't it be the following, just like above?

```suggestion
        CloseableHttpResponse response = http.execute(addXAuthorizationToRequest(request));
```

Reply:
Yes, looks more consistent.